crates/aptos-api-tester/src/consts.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright © Aptos Foundation
+
+use once_cell::sync::Lazy;
+use std::{env, time::Duration};
+use url::Url;
+
+// Faucet constants
+
+pub static DEVNET_NODE_URL: Lazy<Url> =
+    Lazy::new(|| Url::parse("https://fullnode.devnet.aptoslabs.com").unwrap());
+
+pub static DEVNET_FAUCET_URL: Lazy<Url> =
+    Lazy::new(|| Url::parse("https://faucet.devnet.aptoslabs.com").unwrap());
+
+pub static TESTNET_NODE_URL: Lazy<Url> =
+    Lazy::new(|| Url::parse("https://fullnode.testnet.aptoslabs.com").unwrap());
+
+pub static TESTNET_FAUCET_URL: Lazy<Url> =
+    Lazy::new(|| Url::parse("https://faucet.testnet.aptoslabs.com").unwrap());
+
+pub const FUND_AMOUNT: u64 = 100_000_000;
+
+// Persistency check constants
+
+// How long a persistent check runs for.
+pub static PERSISTENCY_TIMEOUT: Lazy<Duration> = Lazy::new(|| {
```

Comment:
I think this is easier to set up with GCP since we can directly import secrets (endpoint, tokens) as environment variables. Would be interested in talking about how it could be done alternatively

Reply:
Nah alright that sounds good, thanks for the context.